features/step_definitions/user_steps.rb

```diff
@@ -168,6 +168,14 @@ def log_in_as(username)
   click_on('I Agree')
 end
 
+And('they choose unsigned assertions') do
+  all('input[type=checkbox][value="signAssertions"]').each do |checkbox|
+    if checkbox.checked? then 
```

Comment:
Is this Capybara? Could we use `uncheck` and lose the `if`?

Reply:
This has surfaced an embarassing html bug in stub-idp... I'll be right back!